### DISCUSSION TITLE
黑田震撼彈日圓急升2.9% 亞股沈淪 專家看法出爐

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#918] [General]
日幣前天忽然升值，央行將10年期的公債殖利率調升，因為前一陣子日圓很便宜就有很多借日圓去投資的，還有去日本買房的投資客，應該都很煩惱，我覺得會反應這麼大，是因為大家都沒有想到，畢竟日本的貨幣寬鬆政策已經持續了很長一段時間，這次也造成台股大跌。
網址：https://tw.stock.yahoo.com/news/黑田震撼彈日圓急升-29-亞股沈淪-專家看法出爐-053427737.html

**Comment by CPCpyt** [comment]
我覺得這是很棒的議題，近期全球的總體環境本來就不穩定，
因此在操作上都要注意風險，很容易一則新聞就能帶來巨大損失。
